route/auth.js

```diff
@@ -1,8 +1,24 @@
 const Router   = require('express').Router;
 const router   = new Router();
 const request  = require('request');
+const jwt = require('../lib/jwt');
 
 
+function checkType(req, res, next) {
```

Comment:
I think we have a wrapper around auth0 to make requests instead of doing it manually